register-istanbul.js

```diff
@@ -9,8 +9,8 @@ var writeOnExit = coverageVar == null ? true : null;
 coffeeCoverage.register({
     instrumentor: 'istanbul',
     basePath: process.cwd(),
-    exclude: ['/test', '/node_modules', '/.git'],
+    exclude: ['/node_modules', '/.git', '/documentation', '/examples'],
```

Comment:
Well, this is my personal preference.. It's that the code coverage of test files should be available too.

That's because there have been countless times when the new-written tests are not correctly run in my development.. Under aggressive development things progress at a rapid pace, so it's be difficult to be careful always to every detail.

Also, because coverage information can be provided as a no additional manual effort, this can be useful for time-effective just hurried-up investigation to fix something wrong as much quick as possible.

Under that situation, just seeing if coverages are intentionally collected is just a plus, I think. :) Also, as a developer, while doing TDD, it's always fun to see my laboured code painted as green regardless of application or test code.

Anyway, that's my two cents. I'm persuading no one. Before the wip status got removed, I'll revert this change. :)